README.md

```diff
@@ -22,86 +22,67 @@ The guide is meant to be a simple starting-point for anyone looking to build Bit
 
 The guide is still being created and should be seen as an evolving resource. Similar to the iterative design process, it starts with core parts then grows in ambition based on community support.
 
-Here is an initial outline that will be updated as needed, content that is live is linked:
+Here is an outline of the content that is currently live:
```

Comment:
Do we even need to repeat this table of contents here? Can we remove it?

Reply:
ACK for removing it.

Reply:
Good point. I guess we can remove it.
Alternative we just keep the top level list with links to the chapters?

Reply:
I have no strong opinion either way. Feel free to decide and then we can merge this update.

Reply:
Removed everything but the chapter links.